fixbackend/billing_information/schemas.py

```diff
@@ -40,3 +47,110 @@ def from_model(billing_entry: BillingEntry) -> "BillingEntryRead":
             period_end=billing_entry.period_end,
             nr_of_accounts_charged=billing_entry.nr_of_accounts_charged,
         )
+
+
+@deprecated("Use PaymentMethodV2 union instead")
+class PaymentMethodV1(str, Enum):
+    AWS = "aws_marketplace"
+
+
+class AwsSubscription(BaseModel):
+    method: Literal["aws_marketplace"]
+    subscription_id: SubscriptionId = Field(description="AWS Marketplace subscription identifier")
+
+
+class NoPaymentMethod(BaseModel):
+    method: Literal["none"]
+
+
+PaymentMethodV2 = Union[AwsSubscription, NoPaymentMethod]
```

Comment:
consider to name it just PaymentMethod

Reply:
Let's not, name collisions are already bad here :)